repository/src/main/java/com/epam/esm/repository/builder/TagQueries.java

```diff
@@ -2,17 +2,11 @@
 
 public final class TagQueries {
 
-    public static final String SELECT_ALL_TAGS = "SELECT id, name FROM tag";
+    public static final String SELECT_ALL_TAGS = "SELECT tags FROM Tag tags";
 
-    public static final String SELECT_TAG_BY_ID = "SELECT id, name FROM tag WHERE id = :id";
+    public static final String SELECT_TAG_BY_NAME = "SELECT tags FROM Tag tags WHERE tags.name = ?1";
 
-    public static final String SELECT_TAG_BY_NAME = "SELECT id, name FROM tag WHERE name = :name";
-
-    public static final String SELECT_TAG_BY_CERTIFICATE = "SELECT id, name FROM tag INNER JOIN gift_certificate_has_tag AS gt ON tag.id = gt.tag_id WHERE gt.gift_certificate_id = :certificate_id";
-
-    public static final String INSERT_TAG = "INSERT INTO tag (name) VALUES (:name)";
-
-    public static final String DELETE_TAG = "DELETE FROM tag WHERE id = :id";
+    public static final String SELECT_TAG_BY_CERTIFICATE = "SELECT c FROM Certificate t INNER JOIN t.tags c where t.id = ?1";
```

Comment:
I may have forgotten a little. What is "?1" at the end?

Reply:
"?1" is for setting a parameter in JPQL `entityManager.createQuery(...).setParameter(1, ...)`